common/version.go

```diff
@@ -31,6 +31,7 @@ import (
 	"strings"
 )
 
+const CloudfuseReleaseURL = "https://api.github.com/repos/Seagate/cloudfuse/releases"
 const CloudfuseListContainerURL = "https://blobfuse2.blob.core.windows.net/release"
```

Comment:
Should we remove these two lines, since they are for blobfuse2. Maybe could be a second PR?

Reply:
Yeah, this neighborhood needs cleaning up. I'll see what I can remove without starting too much trouble.

Reply:
Ok, just did that.